tests/unit/algos/list/delimiterMaster.js

```diff
@@ -2,12 +2,12 @@
 
 const assert = require('assert');
 const DelimiterMaster =
-    require('../../../../lib/algos/list/delimiterMaster').DelimiterMaster;
+    require('../../../../index').algorithms.list.DelimiterMaster;
```

Comment:
I found the issue, it actually exposed a bug that'd happen to the API consumer, because of mixed imports.